src/components/molecules/TabMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import TabItem from './atoms/TabItem';
+
+interface ITabMenuProps {
+  className?: string;
+  tabLabels: Array<string>;
+}
+
+const TabMenu: React.FC<ITabMenuProps> = ({className, tabLabels, children}) => {
+  const [activeTab, setActiveTab] =  useState(0);
+
+  const handleTabClick = (index: number) => setActiveTab(index);
+
+  const renderTabs = () => (
+    tabLabels.map((e, i) => (
+      <TabItem key={e} tabIndex={i} onClick={handleTabClick} isActive={i === activeTab}>{e}</TabItem>),
+      )
+  );
+  return (
+    <React.Fragment>
```

Comment:
This can be shortened down to `<> ... </>`. No difference, just quicker to type.

Reply:
Actually it's not needed here as there's only one child.

Reply:
> This can be shortened down to `<> ... </>`. No difference, just quicker to type.
ik. I've just adopted the convention of writing it explicitly based on Reacts note about many [tools still missing support for this syntax.](https://reactjs.org/docs/fragments.html#short-syntax)

I'm flexible so, I can adopt the short syntax on this project.

Also, indeed not needed here. I'll remove it 😄

Reply:
Aha, I see. In the end it doesn't really matter for the product, but I thought I'd make you aware of it as it's a bit more comfortable to type. 

On the editor support I believe most editors should be able to display them correctly now. I for sure know that `vim`, VSCode, and Sublime Text display them as intended. From what I gathered from the article you linked, it is also more broadly supported when using Typescript compared to Javascript, so that's nice ;)